samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/storage.c

```diff
@@ -0,0 +1,166 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ble_mesh.h"
+#include "device_composition.h"
+#include "storage.h"
+
+static u8_t STORAGE_ID;
+u8_t reset_counter;
+
+static void save_reset_counter(void)
+{
+	char buf[4];
+
+	settings_str_from_value(SETTINGS_INT8, &reset_counter, buf,
+				sizeof(buf));
+
+	settings_save_one("ps/1", buf);
```

Comment:
You might want to use something more friendlier than numbers for the paths, e.g. "ps/rc" (for reset counter). Why "ps" btw? If that's intended to be short for "persistent storage" then it seems a bit unnecessarily generic. Wouldn't it be better to use something specific to the data that's stored under it, e.g. "onoff" or something like that?

Reply:
Here I've change naming style from number to words.
Keep "ps" as it is.